NARFO_BE/NARFO_BE/Controllers/MembersController.cs

```diff
@@ -2,121 +2,133 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Cors;
 using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
 using NARFO_BE.Models;
 
 namespace NARFO_BE.Controllers
 {
-    [Route("api/[controller]")]
-    [ApiController]
+    [Route("Member")]
+    [EnableCors("MyPolicy")]
     public class MembersController : ControllerBase
     {
         private readonly narfoContext _context;
-        List<Members> members = new List<Members>();
+        List<_Member> members = new List<_Member>();
+        public UserManager<_Member> UserManager { get; private set; }
 
         public MembersController(narfoContext context)
         {
             _context = context;
             if (!_context.Members.Any())
             {
-                Members newMember = new Members();
-                newMember.LastName = "admin";
+                _Member newMember = new _Member();
+                newMember.SURNAME = "admin";
                 newMember.Username = "admin";
-                newMember.FirstName = "admin";
-                newMember.Password = "admin";
+                newMember.Firstname = "admin";
+                newMember.Email = "admin@admin.com";
+               
                 _context.Members.AddAsync(newMember);
                 _context.SaveChanges();
             }
         }
-        
-        public void ListofMembers(List<Members> members)
-        {
-            this.members = members;
-        }
-        
 
-        // GET: api/Members
-        [HttpGet]
-        public async Task<ActionResult<IEnumerable<Members>>> GetMembers()
-        {
-            return await _context.Members.ToListAsync();
-        }
-        public IEnumerable<Members> GetAllMembers()
+        public void ListofMembers(List<_Member> members)
         {
-            return members;
+            this.members = members;
         }
 
-        // GET: api/Members/5
-        [HttpGet("{id}")]
-        public async Task<ActionResult<Members>> GetMembers(int id)
+        //
+        // POST: /Account/Login
+        [HttpPost("get/login")]
+        public async Task<ActionResult <_Member>> Login([FromBody] _Member model)
         {
-            var members = await _context.Members.FindAsync(id);
+            
+            _Member user = await _context.Members.FirstOrDefaultAsync(member=>member.Email == model.Email && member.Password == encryption.ComputeHash(model.Password));
```

Comment:
This is just a comment

but it is better to use 

```
bool userExists = await _context.Members.AnyAsync(member=>member.Email == model.Email && member.Password == encryption.ComputeHash(model.Password));
return userExists ? Ok(new { status = "Success", member = model }) : BadRequest(new { status = "Failed", message = "Invalid login" })
```

It is way quicker and doesn't take the full user and put it in an object, coz you not gonna use that user at all
soo what if the user has a lot of information it was gonna be a waste of memory

> Just saying 😃

Reply:
i would not recommend turning it in to a ternary expression, although it might be less code to write it is less readable. there also almost no difference between a ternary expression and an if. Code that is easier to understand is better than code that is just more convenient to type.

Reply:
so you guys are just suggesting your views??